charts/fleet/values.yaml

```
@@ -90,3 +90,12 @@ leaderElection:
   leaseDuration: 30s
   retryPeriod: 10s
   renewDeadline: 25s
+
+## Fleet controller configuration
+controller:
+  reconciler:
+    # The number of workers that are allowed to each type of reconciler
+    workers:
+      gitrepo: "1"
+      bundle: "2"
+      bundledeployment: "3"
```

```suggestion
      bundledeployment: "1"
```

I'd default them all to 1, I think that's the upstream default, too.

---

Of course, I forgot to reset them !